internal-gateway/deployment.yaml

```diff
@@ -36,25 +36,50 @@ spec:
               topologyKey: "kubernetes.io/hostname"
       containers:
        - name: internal-gateway
-         image: "{{ internal_gateway_image.image }}"
+         image: "{{ global.docker_prefix }}/envoy:v1.22.3"
+         command:
+           - /usr/local/bin/envoy
+           - --config-path
+           - /etc/envoy/envoy.yaml
+           - --concurrency
+           - "2"
+           - --service-cluster
+           - gateway
+           - --service-node
+           - $(hostname)
          resources:
            requests:
              cpu: "20m"
              memory: "200M"
            limits:
              cpu: "1"
              memory: "1G"
+         env:
+         - name: ENVOY_UID
```

Comment:
Why do we need to be the root user?

Reply:
This was to be able to bind to port 80, but now that I'm thinking about it, we might not need the envoy to be listening on port 80 on the pod. I'll try an unprivileged port and try to get the load balancer service to send port 80 to the unprivileged port on the {internal-}gateway pod.